<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSettingClass</class>
 <widget class="QDialog" name="DlgSettingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>516</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioStep">
        <property name="text">
         <string>Single Step</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioPC2PC">
        <property name="text">
         <string>PC - PC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioPC2Human">
        <property name="text">
         <string>PC - Human</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioHuman2Human">
        <property name="text">
         <string>Human - Human</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Start color</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioWhite">
        <property name="text">
         <string>White</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioBlack">
        <property name="text">
         <string>Black</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Language (Activate next time)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QComboBox" name="cbLanguage">
        <item>
         <property name="text">
          <string>English</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>中文</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="gbAlgorithm">
     <property name="title">
      <string>Algorithm</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="radioMinMax">
        <property name="text">
         <string>MinMax</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioAlphaBeta">
        <property name="text">
         <string>Alpha-Beta pruning</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioAlphaBetaImproved">
        <property name="text">
         <string>Improved Alpha-Beta pruning</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QGroupBox" name="gbEstimation">
     <property name="title">
      <string>Estimation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QRadioButton" name="radioBasicEstimation">
        <property name="text">
         <string>Basic estimation</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioImprovedEstimation">
        <property name="text">
         <string>Improved estimation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbCacheEstimation">
        <property name="text">
         <string>Cache Estimation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="gbDepth">
     <property name="title">
      <string>Algorithm Parameters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="radioLimitByDepth">
          <property name="text">
           <string>Limit By Depth</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="labelDepth">
            <property name="text">
             <string>Depth</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderDepth">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioLimitByTime">
          <property name="text">
           <string>Limit By Time</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="labelTimeLimit">
            <property name="text">
             <string>Time limit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderTimeLimit">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>60</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>30</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="labelMemory">
          <property name="text">
           <string>Memory Limit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderMemory">
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="pageStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>500</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgSettingClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgSettingClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioLimitByDepth</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelDepth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioLimitByDepth</sender>
   <signal>toggled(bool)</signal>
   <receiver>sliderDepth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioLimitByTime</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelTimeLimit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioLimitByTime</sender>
   <signal>toggled(bool)</signal>
   <receiver>sliderTimeLimit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>483</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
